[{"lineNumber": 6, "col_offset": 15, "nodeName": "TestCase", "type": "Type[unittest.case.TestCase]"}, {"lineNumber": 15, "col_offset": 46, "nodeName": "cm", "type": "unittest.case._AssertRaisesContext[SystemExit]"}, {"lineNumber": 18, "col_offset": 15, "nodeName": "code", "type": "int"}, {"lineNumber": 15, "col_offset": 13, "nodeName": "assertRaises", "type": "Callable"}, {"lineNumber": 15, "col_offset": 31, "nodeName": "SystemExit", "type": "Type[SystemExit]"}, {"lineNumber": 16, "col_offset": 12, "nodeName": "main", "type": "Callable[..., None]"}, {"lineNumber": 18, "col_offset": 15, "nodeName": "exception", "type": "SystemExit"}, {"lineNumber": 15, "col_offset": 13, "nodeName": "self", "type": "TestMain"}, {"lineNumber": 18, "col_offset": 15, "nodeName": "cm", "type": "unittest.case._AssertRaisesContext[SystemExit]"}]